README.md

```diff
@@ -1,76 +1,208 @@
-| **[memdown](https://github.com/level/memdown)** | [![dependencies](https://david-dm.org/Level/memdown.svg)](https://david-dm.org/level/memdown)| [Object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object) |
-| **[redisdown](https://github.com/hmalphettes/redisdown)** | [![dependencies](https://david-dm.org/hmalphettes/redisdown.svg)](https://hmalphettes/redisdown)| [Redis](https://redis.io/) |
-| **[rocksdb](https://github.com/level/rocksdb)** | [![dependencies](https://david-dm.org/Level/rocksdb.svg)](https://david-dm.org/level/rocksdb)| [RocksDB](https://github.com/facebook/rocksdb/) |
-| **[sqldown](https://github.com/calvinmetcalf/sqldown)** | [![dependencies](https://david-dm.org/calvinmetcalf/sqldown/status.svg)](https://david-dm.org/calvinmetcalf/sqldown)| [sqlite3](https://www.npmjs.com/package/sqlite3), [pg](https://www.npmjs.com/package/pg), [mysql](https://www.npmjs.com/package/mysql) |
-
-AbstractLevelDOWN Adapters
-------------
-
-These are libraries that adapt another **[AbstractLevelDOWN](https://github.com/level/abstract-leveldown)** and can be used by **[LevelUP](https://github.com/level/levelup)** 
-
-| name | dependencies | description |
-| ---- | ------------ | ----------- |
-| **[encoding-down](https://github.com/level/encoding-down)**   | [![dependencies](https://david-dm.org/Level/encoding-down.svg)](https://david-dm.org/level/encoding-down) | provides key/value encoding
-
-LevelUP Adapters and Plugins
-------------
-These are libraries that add functionality to **[LevelUP](https://github.com/level/levelup)** 
-
-| name | dependencies | description |
-| ---- | ------------ | ----------- |
-| **[levelup-async-iterator](https://github.com/MeirionHughes/levelup-async-iterator)**   | [![dependencies](https://david-dm.org/MeirionHughes/levelup-async-iterator.svg)](https://david-dm.org/MeirionHughes/levelup-async-iterator) | expose `iterator()` with `[symbol.asyncIterator]`
-
-General
-------------
-
-These are general libraries helpful in using other Level related libraries
-
-| name | dependencies | description |
-| ---- | ------------ | ---- |
-| **[level](https://github.com/level/level)** | [![dependencies](https://david-dm.org/Level/level.svg)](https://david-dm.org/level/level)| convenience package that bundles **[LevelUP](https://github.com/level/levelup)** and  **[LevelDOWN](https://github.com/level/leveldown)**
-| **[level-iterator-stream](https://github.com/level/iterator-stream)** | [![dependencies](https://david-dm.org/Level/iterator-stream.svg)](https://david-dm.org/level/iterator-stream) | Turn a AbstractDown iterator into a readable stream
-
-Other
-------------
+# Awesome Level ![level badge][level-badge]
+
+> A curated list of amazingly awesome **[Level]** modules and resources. [Add yours!](#contributing)
+
+## Core
```

Comment:
Does this header make sense?

Reply:
maybe drop the the "amazingly". Other than that its fine.

Reply:
Sorry I was referring to "Core". I like "amazingly awesome" :)